airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml

```diff
@@ -0,0 +1,18 @@
+acceptance_tests:
```

Comment:
I think this file is missing the line `connector_image: airbyte/source-retently:dev`

Reply:
Added